src/Storage/Device.php

```diff
@@ -100,6 +106,17 @@ abstract public function move(string $source, string $target): bool;
      * @return bool
      */
     abstract public function delete(string $path, bool $recursive = false): bool;
+    
+    /**
+     * Delete file in given path return true on success and false on failure.
```

Comment:
Is it file, directory or both? 🤔

Reply:
it is a directory, updated the description